packages/cysync-core/src/utils/error/index.ts

```diff
@@ -27,6 +30,14 @@ export * from './types';
  */
 const identifyServerErrors = (error: any) => {
   if (error?.isAxiosError) {
+    if (error.response?.data?.code) {
```

Comment:
Is this condition sufficient? can't any other api provide a response with similar structure?

Reply:
Will pick in a different PR